examples/anonymous-objects/anonymous_objects.description

```diff
@@ -1,3 +1,3 @@
 // Object types can be defined in-line as well. These types do not have a type name associated with them.
-// Such anonymous object types can be used in instances where there is no need to refer to the object type by its name
-// (e.g., objects as record or object fields, objects as function parameters). 
\ No newline at end of file
+// Such anonymous object types can be used in instances in which there is no need to refer to the object type by its name
+// (e.g., objects as records or object fields or objects as function parameters). 
```

Comment:
This change is incorrect.

I believe what we are trying to say here is something like 
```suggestion
// (e.g., objects as fields of records or objects or objects as function parameters).

```